src/index.ts

```diff
@@ -823,7 +823,7 @@ app.whenReady().then(async () => {
   }
 
   if (config.get('options.hideMenu') && !config.get('options.hideMenuWarned')) {
-    dialog.showMessageBox(mainWindow, {
```

Comment:
In this code, the result of the call to `showMessageBox` does not matter, so the `await` keyword is not required.